rootfs/etc/cont-init.d/08-svc-snmptrapd.sh

```diff
@@ -42,6 +42,6 @@ mkdir -p /etc/services.d/snmptrapd
 cat >/etc/services.d/snmptrapd/run <<EOL
 #!/usr/bin/execlineb -P
 with-contenv
-/usr/sbin/snmptrapd -f -m ALL -M /opt/librenms/mibs:/opt/librenms/mibs/cisco udp:162 tcp:162
+/usr/sbin/snmptrapd -f -m ALL -M /opt/librenms/mibs:/opt/librenms/mibs/cisco${SNMP_EXTRA_MIB_DIRS} udp:162 tcp:162
```

Comment:
Should this prepend `:` if the variable is not empty?

Reply:
Not sure if this was going to cause an issue or not. I put in the readme example to include the : before the dir, if this is sufficient

Reply:
Put the ':' back in and tested with both variable with string and empty - both worked. Should be good to go